packages/Discretization/src/DTK_PointSearch_def.hpp

```diff
@@ -122,12 +126,47 @@ void buildTopo( unsigned int const n_imports,
             }
         } );
     Kokkos::fence();
+
+#if HAVE_DTK_DBC
+    // The sum of the values in topo_size should be equal to the size of
+    // imported_cell_indices
+    Kokkos::View<unsigned int[DTK_N_TOPO], DeviceType> topo_size_sum(
+        "topo_size_sum" );
+    exclusivePrefixSum( topo_size, topo_size_sum );
+    DTK_REQUIRE( lastElement( topo_size_sum ) == n_imports );
+#endif
+}
+
+template <typename DeviceType>
+void checkOffsetOverflow( Kokkos::View<unsigned int *, DeviceType> offset )
```

Comment:
Did you actually have a test case overflow?

Reply:
No